CHANGELOG.md

```diff
@@ -199,11 +207,12 @@ remote storage on `abq start`:
       ```
 
       Where
-        - `<mode>` is either "store" or "load", depending on whether the file should be stored
+
+      - `<mode>` is either "store" or "load", depending on whether the file should be stored
         into the remote location, or loaded from the remote location.
-        - `<file-type>` is either "manifest" or "results".
-        - `<run-id>` is the run ID of the test suite run.
-        - `<local-path>` is the path to the file on the local filesystem. If the mode is "store",
+      - `<file-type>` is either "manifest" or "results".
+      - `<run-id>` is the run ID of the test suite run.
+      - `<local-path>` is the path to the file on the local filesystem. If the mode is "store",
```

Comment:
Again, prettier. Not sure if this one is correct or incorrect though \cc @ayazhafiz

Reply:
I looked at the render and this seems good. I don't think we have prettier configured on this repo though maybe we should.

Reply:
Yea, sorry it's in my editor 😅 I didn't even realize it made these changes until I looked at the diff